documentation/src/main/asciidoc/topics/con_xsite_auto_offline.adoc

```diff
@@ -1,58 +1,65 @@
+error messages and consumes resources.
 
-To automatically take sites offline, you can then configure the
-number of **consecutive** failures that can occur.
-
-For example, the **NYC** backup configuration specifies five as the number of
-failures after which **NYC** goes offline. If **LON** attempts five consecutive
-backup operations that fail, {brandname} automatically takes **NYC** offline.
-**LON** then stops backing up to **NYC** until you bring the site back online.
+.Timeout for backup operations
+Backup configurations include timeout values for operations to replicate data.
+If operations do not complete before the timeout occurs, Infinispan records
+them as failures.
 
 [source,xml,options="nowrap",subs=attributes+]
 ----
-<backup site="NYC" strategy="ASYNC">
-	<take-offline after-failures="5"/>
+<backup site="NYC" strategy="ASYNC" timeout="10000"> <1>
+	...
 </backup>
 ----
 
-You can also specify the amount of time to wait before taking sites offline.
-When backup operations fail, {brandname} waits before taking sites offline. If
-a backup request succeeds before the wait time runs out, {brandname} does not
-take the site offline.
+<1> Operations to replicate data to **NYC** are recorded as failures if they do
+not complete after 10 seconds.
+
+.Number of failures
+You can specify the number of **consecutive** failures that can occur before
+backup locations go offline.
+
+For example, the following configuration for **NYC** sets five as the number of
+failed operations before it goes offline:
 
 [source,xml,options="nowrap",subs=attributes+]
 ----
-<backup site="NYC" strategy="ASYNC">
-	<take-offline after-failures="5"
-                min-wait="10000"/>
+<backup site="NYC" strategy="ASYNC" timeout="10000">
+	<take-offline after-failures="5"/> <1>
 </backup>
 ----
 
-In the preceding example, if failures occur for 5 consecutive operations,
-{brandname} waits 10 seconds and, if no requests are successful within the 10
-second wait time, {brandname} then takes **NYC** offline.
+<1> If a cluster attempts to replicate data to **NYC** and five consecutive
+operations fail, {brandname} automatically takes the backup offline.
 
-To use only a minimum wait time for automatically taking locations offline, set
-a negative or zero value for the `after-failures` attribute; for example:
+.Time to wait
+You can also specify how long to wait before taking sites offline when backup
+operations fail. If a backup request succeeds before the wait time runs out,
+{brandname} does not take the site offline.
 
 [source,xml,options="nowrap",subs=attributes+]
 ----
-<backup site="NYC" strategy="ASYNC">
-	<take-offline after-failures="-1"
-                min-wait="10000"/>
+<backup site="NYC" strategy="ASYNC" timeout="10000">
+	<take-offline after-failures="5"
+                min-wait="10000"/> <1>
 </backup>
 ----
 
+<1> If five consecutive operations fail, {brandname} waits 10 seconds and, if
```

Comment:
infinispan doesn't wait after the 5 failures. it always waits from the first failure
"the site is taken offline if +5 consecutive failures & 10 seconds elapsed since the first failure"